src/Setup/DevDivVsix/Directory.Build.targets

```diff
@@ -1,5 +1,5 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project>
-  <Import Project="..\..\..\Directory.Build.targets" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
```

Comment:
Why do we need this set of changes?

Reply:
This is not strictly required, but it unifies all cases where a nested **Directory.Build.targets** is provided to import **PackageProject.targets** (previously the `Import` line varied based on file location, so copy/paste was potentially problematic).